linked_list/cpp/linked_list.cpp

```diff
@@ -0,0 +1,204 @@
+#include < iostream >
+using namespace std;
+
+struct node {
+    int data;
+    node * next;
+};
+
+class linked_list {
+    node * head, * tail;
+    public:
+        linked_list() //constructor
+    {
+        head = NULL;
+        tail = NULL;
+    }
+    void create_node(int);
+    void insert_start(int);
+    void insert_end(int);
+    void insert_pos(int, int);
+    void insert_after_pos(int, int);
+    void display();
+    void count_node();
+    void unsort_search(int);
+    void sort();
+    void del_after_node(int);
+    void concatenate(linked_list);
+};
+
+void linked_list::create_node(int value) {
+    node * temp = new node;
+    temp - > data = value;
```

Comment:
{ should almost always be at the end of the previous line  [whitespace/braces] [4]

*Origin: CPPLintBear, Section: `all.cpplint`.*